api-spec/relayapi_messages.yaml

```diff
@@ -1,7 +1,7 @@
 # /relay/v1/messages/{pubsubTopic}:  # Note the plural in messages
 get: # get_waku_v2_relay_v1_messages
   summary: Get the latest messages on the polled topic
-  description: Get a list of messages that were received on a subscribed PubSub topic after the last time this method was called.
+  description: Get a list of messages that were received on a subscribed PubSub topic after the last time this method was called. Messages are selected from the last 50 messages received by the node.
```

Comment:
`--rest-relay-cache-capacity`  applies here, and the same cache instance is used btw.